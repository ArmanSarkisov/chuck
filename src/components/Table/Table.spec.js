import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Table } from './index';

describe('Table test', () => {
  let component;
  const mockData = [{ joke: 'text', id: 1 }];

  beforeEach(() => {
    component = shallow(<Table data={mockData} />);
  });

  it('should render Joke component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
