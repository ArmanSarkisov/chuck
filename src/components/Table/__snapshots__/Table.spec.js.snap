// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table test should render Joke component 1`] = `
<div
  className="flex flex-col"
>
  <div
    className="-my-2 overflow-x-auto"
  >
    <div
      className="py-2 align-middle inline-block min-w-full"
    >
      <div
        className="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg"
      >
        <table
          className="min-w-full divide-y divide-gray-200"
        >
          <Thead />
          <Tbody
            data={
              Array [
                Object {
                  "id": 1,
                  "joke": "text",
                },
              ]
            }
            favorites={Array []}
            handleAddJokeToFav={[Function]}
            user={Object {}}
          />
        </table>
      </div>
    </div>
  </div>
</div>
`;
